util-hadoop/src/main/java/com/google/cloud/hadoop/util/RefreshTokenAuth2Provider.java

```diff
@@ -0,0 +1,230 @@
+package com.google.cloud.hadoop.util;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonToken;
+import com.google.common.flogger.GoogleLogger;
+import com.google.common.io.BaseEncoding;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.InetSocketAddress;
+import java.net.Proxy;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.http.HttpHeaders;
+
+public class RefreshTokenAuth2Provider implements AccessTokenProvider {
```

Comment:
I don't see usages of `RefreshTokenAuth2Provider` in the connector ? 

How does the AuthProvider get wired into the connector ? Are we missing something ?

Reply:
You can define this class using the config `fs.gs.auth.access.token.provider.impl`

That's how personally we wired this provider.

Reply:
Thanks! Can you update `configuration.md` and `changes.md` to reflect the changes ?